modules/apps/portal-scripting/portal-scripting-groovy/src/test/java/com/liferay/portal/scripting/groovy/internal/GroovyScriptingExecutorTest.java

```diff
@@ -43,6 +47,25 @@ public void testBindingInputVariables() throws Exception {
 			Collections.emptySet(), "binding-input");
 	}
 
+	@Test
+	public void testGroovyRuntimeExceptionSerialization() throws Exception {
```

Comment:
@marianoalvarosaiz Could you clarify in the commit message that this commit adds a test to reveal the issue?